modules/apps/object/object-test/src/testFunctional/tests/ObjectFields.testcase

```diff
@@ -1489,6 +1489,103 @@ definition {
 		}
 	}
 
+	@description = "LPS-164948 - Verify that the user is able to add the Formula Field when managing Custom Validation"
+	@priority = 4
+	test CanManageFormulaFieldWithCustomValidation {
+		property custom.properties = "feature.flag.LPS-164948=true";
+
+		task ("Given: an Object with Formula Field") {
```

Comment:
We want to be descriptive here so that we understand exactly what the task is doing. 

`Given a custom object with two integer fields and a formula field summing the integer field values`

Reply:
Done !